pkg/disk/directio_windows.go

```diff
@@ -0,0 +1,44 @@
+// +build windows
```

Comment:
Adding this (and fixing the build+missing flag) makes the server completely unusable.

```
λ warp mixed -duration=2m -obj.randsize
...
Mixed operations.                                                                                
                                                                                                 
Operation: DELETE                                                                                
 * Operations: 15 (+Inf% of operations)                                                          
                                                                                                 
Throughput by host:                                                                              
 * http://127.0.0.1:9001: Avg: NaN obj/s (0s, starting 00:00:00 UTC)                             
 * http://127.0.0.1:9002: Avg: 58.43 obj/s (51ms, starting 10:47:46 CEST)                        
 * http://127.0.0.1:9003: Avg: 0.09 obj/s (1m50.924s, starting 10:47:46 CEST)                    
 * http://127.0.0.1:9004: Avg: NaN obj/s (0s, starting 00:00:00 UTC)                             
                                                                                                 
Operation: STAT                                                                                  
 * Operations: 25 (+Inf% of operations)                                                          
                                                                                                 
Throughput by host:                                                                              
 * http://127.0.0.1:9004: Avg: 43.00 obj/s (93ms, starting 10:47:46 CEST)                        
 * http://127.0.0.1:9001: Avg: 353.83 obj/s (7ms, starting 10:47:46 CEST)                        
 * http://127.0.0.1:9002: Avg: 0.02 obj/s (1m28.983s, starting 10:47:46 CEST)                    
 * http://127.0.0.1:9003: Avg: 0.12 obj/s (1m50.934s, starting 10:47:46 CEST)                    
                                                                                                 
Operation: PUT                                                                                   
 * Operations: 17 (+Inf% of operations)                                                          
                                                                                                 
Throughput by host:                                                                              
 * http://127.0.0.1:9001: Avg: 91.38 MiB/s, 25.55 obj/s (115ms, starting 10:47:46 CEST)          
 * http://127.0.0.1:9002: Avg: NaN obj/s (0s, starting 00:00:00 UTC)                             
 * http://127.0.0.1:9003: Avg: 0.10 MiB/s, 0.07 obj/s (1m50.876s, starting 10:47:46 CEST)        
 * http://127.0.0.1:9004: Avg: 0.27 MiB/s, 0.62 obj/s (6.502s, starting 10:49:11 CEST)           
                                                                                                 
Cluster Total:  NaN obj/s (0s, starting 00:00:00 UTC)                                            
 * http://127.0.0.1:9001:  0.10 MiB/s, 0.11 obj/s (1m40.273s, starting 10:47:46 CEST)            
 * http://127.0.0.1:9002:  0.00 MiB/s, 0.10 obj/s (1m40.329s, starting 10:47:46 CEST)            
 * http://127.0.0.1:9003:  0.15 MiB/s, 0.29 obj/s (1m55.469s, starting 10:47:46 CEST)            
 * http://127.0.0.1:9004:  0.08 MiB/s, 0.14 obj/s (1m38.804s, starting 10:47:46 CEST)            
```

Could we get rid of this bonus change for now?

Reply:
I removed the NO_BUFFERING and instead kept it as WRITE_THROUGH